versions.toml

```diff
@@ -0,0 +1,4 @@
+# Update these values when a new release is created.
+# `stacks-common/build.rs` will automatically update `versions.rs` with these values.
+stacks_node_version = "3.1.0.0.2"
+stacks_signer_version = "3.1.0.0.2.0"
```

Comment:
I think from a fresh start I'd agree with you, but since we already have tooling and standards in place to use the STACKS_NODE_VERSION env variable, I feel like it's better to just keep the nomenclature here

Reply:
yeah that makes sense. i don't feel strongly here so I'll resolve.